lib/hypervisor/hv_kvm/monitor.py

```diff
@@ -811,6 +771,21 @@ def GetCpuInformation(self):
 
     return self.Execute("query-cpus-fast")
 
+  @_ensure_connection
+  def GetHotpluggableCPUs(self):
+    """ Get CPU type which could be plugged
+        uses the query-hotpluggable-cpus which does not interrupt the guest
+    """
+
+    return self.Execute("query-hotpluggable-cpus")
+
+  @_ensure_connection
+  def QueryMemorySummary(self):
```

Comment:
same as above, why is this method added here?

Reply:
removed